src/cadenza.js

```diff
@@ -785,6 +807,15 @@ function createParams({
       webApplicationLink: webApplication.externalLinkId,
       webApplicationLinkRepository: webApplication.repositoryName,
     }),
+    ...(filter &&
+      Object.entries(filter).reduce(
```

Comment:
Usually I find Object.fromEntries() easier to read / understand.
```javascript
    ...(filter &&
      Object.fromEntries(
        Object.entries(filter).map(([variable, value]) => [
          `filter.${variable}`,
          JSON.stringify(value instanceof Date ? value.toISOString() : value),
        ]),
      )),
```

Reply:
Right, that is much nicer, I changed it.